src/components/Header/Header.jsx

```diff
@@ -0,0 +1,78 @@
+import Container from '@mui/material/Container';
+import SvgIcon from '@mui/material/SvgIcon';
+import css from './Header.module.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { selectIsModalLogoutOpen } from '../../redux/selectors';
+import { openModalLogout } from '../../redux/auth/auth.slice';
+import { selectUserFirstName } from '../../redux/selectors';
+import Logo from '../Logo/Logo';
+
+const Header = () => {
+  const name = useSelector(selectUserFirstName);
+  const isModalLogoutOpen = useSelector(selectIsModalLogoutOpen);
+  const dispatch = useDispatch();
+
+  const handleModalLogoutOpen = ev => {
+    ev.preventDefault;
+    dispatch(openModalLogout());
+  };
+
+  return (
+    <div className={css.headerWrapper}>
+      <Container
+        sx={{
+          '@media (min-width: 780px)': {
+            padding: '0 32px',
+          },
+          '@media (min-width: 1280px)': {
+            padding: '0 16px',
+          },
```

Comment:
Jak używasz mediów, to trzeba dawać tak:
'@media **screen and** (min-width: 780px)'
Tak samo z desktopem :)

Reply:
Oczywiście :D